### PR TITLE
Add release notes for 6-29-2021 through 7-27-2021

### PR DESCRIPTION
**Pull request by patphongs**
Release notes for 15.1 through 15.3

### CODE REVIEW

File: release_notes/release_notes.md [hunk 1]
```diff
@@ -1,3 +1,44 @@
...
+
+## Changed
+- **Help for candidates and committees:** Added disclaimer examples for separate segregated funds and corporations and labor organizations to relevant pages.
+- **Campaign finance data:** No longer default to 2020 presidential race
+
+## Under the hood
+- In the Production environment, scale up celery worker memory to 1500M(1.5GB) per instance. In the Development reduce the celery worker memory to 512MB
+- Django version upgraded to 3.1.12
+
+
```
Comment (PaulClark2) [inline release_notes/release_notes.md:41]: It looks like we forgot 14 innovation too. 

# June 15, 2021 (PI 14 Innovation)
## Added
- **Legal:** A 2009 Federal Register notice on AO procedures was added to the [Policy and other guidance page](https://www.fec.gov/legal-resources/policy-other-guidance/#rules-of-agency-organization-procedure-or-practice) and the [Guidance search](https://www.fec.gov/legal-resources/policy-and-other-guidance/guidance-documents).

## Changed

- **General:** Banner warning of degraded performance for Internet Explorer will now appear for every version of that browser (previously only for versions < 10)

## Fixed
- **API:** Fixed Schedule-b (Disbursement) e-file export multi same rows bug.
- **General** Fixed scrolling after resizing window while menu was open
- **General:** Fixed minor grammatical errors to three entries.
- **Homepage:** Fixed aspect ratios of Commissioners' photos no longer skew for some browsers on some devices

## Under the hood
- Added redirects from older content to newer content
- Security and stability updates
- Homepage speed improvements
- Increased privacy for all off-domain links

File: release_notes/release_notes.md [hunk 2]
```diff
@@ -1,3 +1,44 @@
...
+- **Legal Resource:** Add additional Administrative fine
+ parameters to /legal/docs/ api  endpoint
+- **API:** Add filters to /totals/entity_type/ to support PAC/Party totals data table
+
+## Fixed
+- **General:** Fixed glossary spacing issues
+- **Campaign finance data:** Fix bad link to OGE reports
+- **API:** Fixed API doc typos for form 1 and form 2
+## Under the hood
+- Add migration file for fec_mur.af_number_rad_civil_pnlty_updt
```
Comment (PaulClark2) [inline release_notes/release_notes.md:26]: Maybe change this to something like, "Added data to support Administrative Fines data table"

File: release_notes/release_notes.md [hunk 3]
```diff
@@ -1,3 +1,44 @@
...
+# June 29, 2021 (15.1) 
+## Added
+- **Legal Resources:** Added AO Procedures 2009 Federal Register notice to [guidance search](https://www.fec.gov/legal-resources/policy-and-other-guidance/guidance-documents/) and to [Policy and other guidance](https://www.fec.gov/legal-resources/policy-other-guidance/)page 
+- **API** Add leadership PAC sponsor names to /committees/ endpoint
+
+## Changed
+- **Help for candidates and committees:** Added disclaimer examples for separate segregated funds and corporations and labor organizations to relevant pages.
+- **Campaign finance data:** No longer default to 2020 presidential race
+
+## Under the hood
```
Comment (PaulClark2) [inline release_notes/release_notes.md:37]: I'd probably change this to something like, 
"Upgraded software packages"
"Increase memory allocation for celery worker"

**Review by PaulClark2 — CHANGES_REQUESTED** [review]
See my in-line suggestions below.

**Comment by patphongs** [comment]
Thanks @PaulClark2, I've incorporated the changes you suggested

**Review by PaulClark2 — APPROVED** [review]
Looks good. Thank you for incorporating my suggestions.

**Pull request merged**